ontology-engine/graph-core_2.11/src/main/scala/org/sunbird/graph/util/CSPMetaUtil.scala

```diff
@@ -0,0 +1,74 @@
+package org.sunbird.graph.util
+
+import org.apache.commons.collections4.MapUtils
+import org.slf4j.LoggerFactory
+import org.sunbird.common.Platform
+import org.sunbird.common.dto.Property
+import org.sunbird.graph.dac.model.Node
+
+import scala.collection.JavaConverters._
+
+object CSPMetaUtil {
+  private[this] val logger = LoggerFactory.getLogger(classOf[CSPMetaUtil])
+  private val defaultMetadataList = java.util.Arrays.asList("appIcon","posterImage","artifactUrl","downloadUrl","variants","previewUrl","pdfUrl")
+  private val defaultWriteBasePath = java.util.Arrays.asList("https://sunbirddev.blob.core.windows.net","https://obj.dev.sunbird.org")
```

Comment:
This path is not common for others(adopters). We should not have this defaultWriteBasePath